### DISCUSSION TITLE
where to find old versions of documentation

### DISCUSSION THREAD

**Discussion by abubelinha** [py-pdf/pypdf#1605] [Q&A] [answered]
I use an up-to-date version of pypdf at home, but I am tied to an old computer at work. 
I cannot upgrade its Python or OS: I have to cope with that for generating some pdf reports.

I just want to find the appropriate source of documentation for my version of the library, so I stop going nuts trying to find out the reasons of my errors (I mostly try to adapt to use at work some scripts I made at home).

## Environment
```
C:\Python34\python -m platform
Windows-XP-5.1.2600-SP3

C:\Python34\python --version
Python 3.4.4

C:\Python34\python -c "import PyPDF2 as pypdf;print(pypdf.__version__)"
1.28.6
```

Thanks

**Comment by pubpub-zz** [comment, marked as answer]
https://pypdf2.readthedocs.io/en/1.28.4/
readthedocs proposes to access old version at bottom of left column

**Reply by abubelinha** [reply]
Thanks. 
That's exactly what I need but I got confused about the so many versions of this package.
